developer-docs/how-to-guide/how_to_create_badge.md

```diff
@@ -16,178 +16,198 @@ This document explains the procedure of creating and awarding badges.
 
->Note: The badge class only needs to be created once for one type of badge. The same badge class can be awarded multiple times to different recipients. 
+It is recommended to create a new badge class for each type of accomplishment which is being recognised. The badge class only needs to be created once for one type of badge. It is not required to create all the badge classes at once. As new types of accomplishments are recognised, new badge classes can be created.
 
 The organization administrator must:
 
-1. Decide a badge name 
+1. Decide  badge name 
 
-2. Criteria for awarding the badge
+1. Set the criteria for awarding the badge
 
-3. Use the issuerId received from the [Create Issuer API](apis/badgingframeworkapi/#operation/CreatePost) call to indicate which entity is awarding the badge
+1. Choose roles in the system who are authorized to award instance of the badge class
 
-4. Choose an image to be associated with the badge. This image will be displayed on the profile page of all the people who receive the badge 
+1. Specify the rootOrgId within which this badge class is defined.
 
-The following is an example of the request body for creating badge class, the sample values provided in the request body are indicative:
+1. Use the issuerId received from the [Create Issuer API](apis/badgingframeworkapi/#operation/CreatePost) call to indicate which entity is awarding the badge
+
+1. Choose an image to be associated with the badge. This image will be displayed on the profile page of all the people who receive the badge 
+
+In our scenario, XYZ Corp Certifications decides that it will issue a badge called Basic Alphabet Expert to employees who complete their basic alphabet training. The criteria for awarding the badge is that the employee can list all the letters of the alphabet. This class of badges will be issued by the users who have the `BADGE_ISSUER` role. The graphic design unit at XYZ Corp has designed a cool emblem for this badge which is present on the administrator's computer.
+
+We will use these details to construct the request body for creating badge class.
 
 #### Request Body
 
+The organisation administrator will execute a request with the following parameter values.
+
 Parameter that are submitted through form ```application/x-www-form-urlencoded```, multipart, formdata, or both are usually used as the content type of the request 
 
-	issuerId:issuerslug-66
-	name: Padma
-	description: Badge Issuer 
-	criteria: Should be created by org admin
-	rootOrgId: 0124758459210711040
-	type: user
-	subtype: badge
-	roles: BADGE_ISSUER
-	images : C:\Users\Pictures\badge_logo.png
+```
```

Comment:
L119-128 which specify contextual parameters need to be converted into the appropriate cURL request. @Sunitashelar please get correctly formatted version from developer. 

cc: @vrayulu @madhucr